test/pageObjects/caseDetailsPage.js

```diff
@@ -64,8 +64,12 @@ class CaseDetailsPage extends BasePage {
    */
 
   async isPrintButtonReady() {
```

Comment:
can we check if this change is needed, does the original implementation of this function not return false if the button is not here?

Reply:
Yes, we need this change.  Otherwise, it errors since the print button location do not exists

Reply:
ok thats fine